File: c#/Empleados/Repartidor.cs
namespace Empleados{

    class Repartidor : Empleado{

        private string zona;
        public Repartidor(string name,int edad,double salario,string zona):base(name,edad,salario){
            if(edad <25 && zona.ToLower() == "zona 3"){
                this.Salario =  this.Salario + this.PLUS; 
            }
            this.zona = zona;
        }

        public override string ToString()
        {
            return base.ToString() + "\nZona: " + this.zona;
        }

    }
   
}

File: typescript/Empleados/Repartidor.ts
import Empelados from "./Empleados";

export default class Repartidor extends Empelados{
    private zona:string

    constructor(nombre:string,edad:Number,salario:number,zona:string){
        
        super(nombre,edad,salario);
        if(edad < 25 && zona.toLowerCase() == "zona 3"){
            this.setSalario(salario + this.PLUS) 
         }
        this.zona = zona
    }

    

}